### DISCUSSION TITLE
Diamond notifications

### DISCUSSION THREAD

**Discussion by lucefx** [deso-protocol/dips#132] [Medium CIPs]
Any chance we can get diamond notifications linked to the post that got tipped, like it was before, please? 

It would be nice if the notification still said 'sent you diamonds' rather than 'sent you $', too.

Actually, can the tip/diamond notifications please just look exactly like they used to before? :)

Sorry, I don't know the proper format to write a CIP 😬

**Comment by tijno** [comment]
@lucefx mae replied on discord the other day that this was unintentional and will be sorted.

![20210819-OlqiimiK](https://user-images.githubusercontent.com/69529928/130032599-41d60585-513b-4a9e-aec8-bf104dcd02df.png)

**Reply by lucefx** [reply]
Thank Tijn! :)

**Comment by superzordon** [comment]
Resolved via https://github.com/bitclout/core/pull/106 https://github.com/bitclout/backend/pull/129 https://github.com/bitclout/frontend/pull/423. @lucefx can we close?

**Reply by lucefx** [reply]
Of course! :)
